pkg/demoinfocs/net_messages.go

```diff
@@ -19,34 +20,30 @@ func (p *parser) handlePacketEntities(pe *msg.CSVCMsg_PacketEntities) {
 
 	r := bit.NewSmallBitReader(bytes.NewReader(pe.EntityData))
 
-	currentEntity := -1
-	for i := 0; i < int(pe.GetUpdatedEntries()); i++ {
-		currentEntity += 1 + int(r.ReadUBitInt())
-
-		cmd := r.ReadBitsToByte(2)
```

Comment:
by replacing this with ReadBit we change the behaviour so that in certain cases the second bit is not read. I remember this caused a bug before which is why ReadBitsToByte(2) was introduced at some point.

Will need to check that this isn't bringing that bug back

Reply:
looks good :+1:

Reply:
Thank you! I also parsed about 500 demos without any issues.